src/pages/MoviePage/styled.jsx

```diff
@@ -0,0 +1,144 @@
+import styled from "styled-components";
+import { ReactComponent as StarIcon } from "../../assets/icons/star.svg";
+
+export const MainWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 64px;
+  align-items: center;
+  background-color: ${({ theme }) => theme.colors.moviePage.background};
+  margin-bottom: 64px;
+
+  @media only screen and (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    margin-bottom: 16px;
+  }
+`;
+
+export const Header = styled.div`
+  background-image: ${({ backdropPath }) => `url(${backdropPath})`};
+  background-size: cover;
+  background-position: center;
+  max-width: 1368px;
+  width: 100%;
+  height: 770px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  box-shadow: inset 8px -3px 100px 74px rgba(0, 0, 0, 1);
+
+  @media only screen and (max-width: ${({ theme }) => theme.breakpoints.l}) {
+    height: 400px;
+  }
+
+  @media only screen and (max-width: ${({ theme }) => theme.breakpoints.xs}) {
+    height: 200px;
+  }
+`;
+
+export const Wrapper = styled.div`
+  max-width: 1368px;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 64px;
+
+  @media only screen and (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    gap: 16px;
+  }
+`;
+
+export const Section = styled.section`
```

Comment:
For cast section, I think grid auto-fill would work better.

Reply:
Thank you for your opinion, but I prefer to leave Flex here. using grid auto fill causes some problems
<img width="821" alt="Zrzut ekranu 2023-11-21 o 18 13 06" src="https://github.com/jakub-bartoszek-developer/Movie-browser/assets/131645596/acede0f6-72b9-432a-ab9e-8fc9cffab9ec">